tools/build/build.js

```diff
@@ -238,6 +238,40 @@ export const DmTarget = new Juke.Target({
   },
 });
 
+export const JustDmTarget = new Juke.Target({
```

Comment:
if it's a full copy of the DM target it may be better to make depending on the cutter conditional via input arg, sort of the inverse of how DmMapsIncludeTarget works.

Reply:
yeah, I don't like writing javascript though